package.json

```diff
@@ -53,5 +66,32 @@
   "homepage": "https://github.com/thebruce/molotov#readme",
   "engines": {
     "node": ">=4"
+  },
+  "jest": {
+    "cacheDirectory": ".tmp/jest_cache",
+    "coverageDirectory": "__coverage__",
+    "coverageReporters": [
+      "json",
+      "json-summary",
```

Comment:
Unless you're using the json stuff somewhere or coveralls uses it I don't think you need it, lcov, text, and text-summary should be sufficient.

Reply:
No, just what is needed in coveralls i think. 👍